IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MechanicalFastenerOverheadContactSystemRigidSupport/DocPropertySet.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MechanicalFastenerOverheadContactSystemRigidSupport" UniqueId="37ce8447-2a11-e7c4-7e22-82827dd949f7" ApplicableType="IfcMechanicalFastener" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="AssemblyInstruction_3W5i6nbkP9NBt_M6PTB5jW" />
+		<DocProperty xsi:nil="true" href="Stagger_1dWD_52DDFheUkLfvK_Wga" />
+	</Properties>
+</DocPropertySet>
+
```

Comment:
Applicable to all PDTs?

Reply:
No! We made a wrong mapping... It's a pset from IfcMember PDT MEMBER. 
Commit : c8245c45a7f488800e3e5c5dd32f498fffa28154